src/main/java/edu/uc/inventorymanager/dto/Item.java

```diff
@@ -1,15 +1,12 @@
 package edu.uc.inventorymanager.dto;
 
 import lombok.Data;
-import lombok.NonNull;
-import lombok.RequiredArgsConstructor;
 
-@RequiredArgsConstructor
 public @Data class Item {
-    @NonNull private int id;
+    private int id;
     private String name;
-    @NonNull private String description;
+    private String description;
```

Comment:
Why remove these annotations?  Does this decrease technical debt?  If so, how?
The group likely put them here for a reason.